src/OpenSearch.Net/Responses/ServerException/ErrorCause.cs

```diff
@@ -113,7 +111,6 @@ internal static class ErrorCauseFormatterStatics
 			{ "index", 6 },
 			{ "index_uuid", 7 },
 			{ "lang", 8 },
-			{ "license.expired.feature", 9 },
```

Comment:
If we remove this line do we know if the number value needs to adjust downstream or is this important that the actual numbers stay as they are?

Reply:
I will test this out. Thanks

Reply:
Updated the changes to adjust the numbers downstream.

Reply:
Based on https://github.com/Bit-Quill/opensearch-net/blob/main/tests/Tests/ClientConcepts/ServerError/ComplexErrorTests.cs, the number values are not used.